go/interpreter/lfvm/instructions_test.go

```diff
@@ -1827,6 +1827,59 @@ func TestOpBlockhash(t *testing.T) {
 	}
 }
 
+func TestInstructions_Sha3_ReportsOutOfGas(t *testing.T) {
+	tests := map[string]struct {
+		size          uint64
+		expectedError error
```

Comment:
Since all cases expect an out of gas error, it should be removed from the test input. Instead of the struct you could then directly save the size.

Reply:
I'll keep in mind for the changes needed after #838  is merged.